### PR TITLE
Change incorrect hash and license

### PR DESCRIPTION
**Pull request by heyjared**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Please [amend](http://stackoverflow.com/a/179147/1661012) the commit message to include the name of the updated cask.

**Comment by jaaaaared** [comment]
Done

**Comment by vitorgalvao** [comment]
Something went wrong. There are now merge conflicts, and you’re changing over two thousand files.

**Comment by jaaaaared** [comment]
Sorry. let me try again with another pull request
